src/main/java/org/mockito/internal/util/MockUtil.java

```diff
@@ -104,10 +131,24 @@ public static boolean isMock(Object mock) {
         if (mock == null) {
             return false;
         }
+        return getMockHandler0(mock) != null;
+    }
+
+    private static MockHandler<?> getMockHandler0(Object mock) {
```

Comment:
Can we use a more descriptive name here? What about `maybeGetMockHandler`?

Reply:
I renamed it to `getMockHandlerOrNull`.